src/components/Progress/Progress.tsx

```diff
@@ -0,0 +1,95 @@
+/*
+Copyright 2024 New Vector Ltd
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React, { forwardRef } from "react";
+import classNames from "classnames";
+
+import styles from "./Progress.module.css";
+
+import { Root, Indicator } from "@radix-ui/react-progress";
+import useId from "../../utils/useId";
+
+type ProgressProps = {
+  /* The size variant of the progress bar */
+  size: "sm" | "lg";
+
+  /* The color variant to use for the progres bar indicator */
+  color?: "red" | "orange" | "lime" | "green";
```

Comment:
What happens if you don't specify a colour?

Also we're a British company, colour*
![image](https://github.com/element-hq/compound-web/assets/2403652/eb3c7e98-8a88-4b13-8ab7-ccd0ec177309)

and `progress*`

Reply:
It becomes gray. You probably should only use this for empty state, as this also propagates to the label on top. See [the password strength story](https://quenting-progress-bar.compound-web.pages.dev/?path=/docs/form-password-form--docs#empty)

And damn I never noticed the britishness of props. But why are the token using `color` then? 🤔 

Also, https://github.com/hashanp/postcss-spiffing

Reply:
> It becomes gray. You probably should only use this for empty state, as this also propagates to the label on top.

Can the default be commented plz

> But why are the token using color then?

No idea

> https://github.com/hashanp/postcss-spiffing

This is awesome lol

Reply:
I'm avoiding the problem by renaming the prop `tint` if you're alright with that 😬 

I also tweaked the comments, and made sure they get parsed as JSDoc comments, so that they render in the IDE and Storybook